shared/src/main/scala/scala/collection/immutable/VectorMap.scala

```diff
@@ -0,0 +1,169 @@
+package scala.collection.immutable
```

Comment:
sorry I missed this before. while it's certainly possible this could eventually become part of the Scala 2.13 collections, for now it's not okay to use `scala.collection`